Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM python:3.7.9
 
 RUN apt-get update && apt-get -y install libpq-dev
+RUN apt-get install -y nodejs
+RUN apt-get install -y npm
```

Comment:
```suggestion
RUN apt-get install -y nodejs=v10.20.1
RUN apt-get install -y npm=6.14.4
```

Reply:
@maxxcrawford I don't - I would generally say "latest" so we avoid security vulnerability issues, but I don't know if there are any version specific dependencies in the application? Sorry I don't really have enough context to say one way or the other. Are there any concerns you have with using `npm`?

Just curious, are we only using the output of `npm install` for use with the django app? If so, maybe we could use a [multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds) and do the `npm` bits in a separate container to keep the final image as small as possible?

Reply:
Also, if you don't care about the minor version being pinned, you could start your multi-stage build with `node:12` (or whatever major version you'd like) which will make sure you're pulling the latest for that major version while still keeping it pinned.

Reply:
(Catching up from offline discussion with @moz-astults) 

We tried the multi-stage build, but node/gulp is required in the same container as the core app. We reverted back to this method, however – instead of `apt-get update`, we use [nodesource](https://github.com/nodesource/distributions/blob/master/README.md#debinstall) to install Node (includes `npm`). We're using `14.x` to get the latest stable in that major version.